combat-trainer.lic

```diff
@@ -1915,11 +1918,16 @@ class AbilityProcess
     @barb_buffs
       .select { |name| Flags["ap-#{name}-expired"] }
       .each do |name|
-        Flags.reset("ap-#{name}-expired")
-        activate_barb_buff(name)
-        echo "Activated barb_buff #{name}!" if $debug_mode_ct
-        waitrt?
-        pause 6 if get_data('spells').barb_abilities[name]['type'].eql? 'meditation'
+        if mana < @barb_buffs_inner_fire_threshold
+          echo "Not activating barb_buff #{name} until have at least #{@barb_buffs_inner_fire_threshold} inner fire" if $debug_mode_ct
+        elsif activate_barb_buff(name)
```

Comment:
❓ Should this now reference `activate_barb_buff?`? If so, can `activate_barb_buff` be removed from DRCA?

Reply:
Yes, now that https://github.com/rpherbig/dr-scripts/pull/4666 got merged I need to update this to use the predicate named method name and remove the old name from DRCA. Thanks